### ISSUE TITLE
Warn when string after ! is anything other than "important"

### ISSUE THREAD

**Issue by jameswilson** [CSSLint/parser-lib#156]
Similar to issue #21. Would be nice to see support for [this IE<=7 hack](http://browserhacks.com/#hack-90ce8e1eeeb5f3d0971df30d412934b4):

```css
.selector { property: value !ie; }
```

Where `!ie` acts as an `!important` and the string after ! can be anything.

Currently it causes a parse error.

    ERROR: Expected RBRACE at line X, col Y. This rule looks for recoverable syntax errors. (errors) Browsers: All

Similarly, sass uses `!default`  so it would be nice to kind of detect for that, instead of blowing up.

**Comment by stubbornella** [comment]
Why not use the * property hack?




.selector { *property: value; }

**Comment by jameswilson** [comment]
You're right that is a valid workaround AND it doesn't throw any parsing errors! But certainly seeing `selector { property: value !ie7andbelow; }` is clearer than the star hack, particularly on aging projects.
